### ISSUE TITLE
not working at all

### ISSUE THREAD

**Issue by rverma-dev** [clemenscodes/nx-plugins#134]
**Describe the bug**

```
npx create-nx-workspace@latest

nx g nx-cmake:init                                                                                                                                                                                                                    37s

>  NX  Generating nx-cmake:init

✔ C or C++? · C++
✔ Where do you want to generate the CMake configuration? · .cmake
✔ Where do you want to generate the global include directory? · include
✔ Where do you want to generate binaries? · bin
✔ Where do you want to generate libraries? · libs
✔ Do you want to generate an opinionated set of configuration files for clang-format and clang-tidy? (Y/n) · false
UPDATE nx.json
CREATE .cmake/settings/set_binary_settings.cmake
CREATE .cmake/settings/set_compiler_settings.cmake
CREATE .cmake/settings/set_global_settings.cmake
CREATE .cmake/settings/set_library_settings.cmake
CREATE .cmake/settings/set_project_settings.cmake
CREATE .cmake/utils/install_cmocka.cmake
CREATE .cmake/utils/install_gtest.cmake
CREATE .cmake/utils/link_cmocka.cmake
CREATE .cmake/utils/link_gtest.cmake
CREATE .cmake/utils/link_shared_library.cmake
CREATE .cmake/utils/link_static_library.cmake
CREATE .cmake/utils/make_var_readonly.cmake
CREATE .cmake/utils/print_variables.cmake
CREATE include/common.h
CREATE include/libcmocka.h
CREATE include/libgtest.h
CREATE CMakeLists.txt


 nx g nx-cmake:b test                    

 >  NX   Failed to process project dependencies with "nx-cmake".

   Cannot destructure property 'projects' of 'context' as it is undefined.
```

**Comment by clemenscodes** [comment]
This will be fixed in release v0.5.0 until next week

**Comment by clemenscodes** [comment]
@rverma-dev the issue should be fixed now using v0.5.0, please let me know if still any issues persist and I will fix them asap.
